projects/manage-supervision-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/service/SentenceService.kt

```diff
@@ -62,20 +63,19 @@ class SentenceService(
                 additionalSentences.map { it.toAdditionalSentence() }
             ),
             order = disposal?.toOrder(),
-            requirements = requirementRepository.getRequirements(id, eventNumber).map { it.toRequirement() },
-            courtDocuments = documentRepository.getCourtDocuments(id, eventNumber).map { it.toCourtDocument() }
+            requirements = requirementRepository.getRequirements(id, eventNumber)
+                .map { it.toRequirement(id, eventNumber) },
+            courtDocuments = documentRepository.getCourtDocuments(id, eventNumber).map { it.toCourtDocument() },
+            disposal?.id?.let { getUnpaidWorkTime(it) }
```

Comment:
Do we also need to check whether they have any unpaid work requirements against the disposal here, and return null if not?

Reply:
line 123 does this check and returns null for this scenario